setup.py

```diff
@@ -68,8 +68,12 @@
     ],
     extras_require={
         'rllib': [
+            'GPUtil',
+            'GPy',
             'gym',
             'ray[rllib,default]==1.13.0',
+            'scikit-learn',
```

Comment:
I don't think this related to this change, and I don't think it's currently needed by the rllib example?

Reply:
It's also needed by rllib to display some statistics